mne/viz/tests/test_ica.py

```diff
@@ -73,6 +73,7 @@ def test_plot_ica_sources():
     """
     import matplotlib.pyplot as plt
     raw = io.Raw(raw_fname, preload=True)
+    raw.crop(0, 1, copy=False)
```

Comment:
I can fix this on my click pr I'm about to push that way I can then rebase and save a CI build